homeassistant/components/binary_sensor/konnected.py

```diff
@@ -0,0 +1,74 @@
+"""
+Support for wired binary sensors attached to a Konnected device.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/binary_sensor.konnected/
+"""
+import asyncio
+import logging
+
+from homeassistant.components.binary_sensor import BinarySensorDevice
+from homeassistant.components.konnected import (DOMAIN, PIN_TO_ZONE)
```

Comment:
We're not allowed to overwrite the binary sensor domain with the `konnected` component domain. Instead import it like so:
```py
from homeassistant.components.konnected import (DOMAIN as KONNECTED_DOMAIN, ...)
```